.travis/checkDoxygenDocumentation.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+return_code=0
+return_code2=0
+return_code3=0
+HOMEPATH=$PWD
+
+
+## We first check that the doxygen.log is empty
+if [ -s doxygen.log ]
+then
+    return_code=0
+else
+    return_code=1
+    echo "Doxygen log file not empty !"
+    cat doxygen.log
+fi
+   
+## We check src code consitency
+cd src/
+if ! $( $HOMEPATH/.travis/check_src_file_tag.sh  )
+then
+    return_code2=1;
```

Comment:
I don't think so since the command is enclosed between brackets (it doesn't work on my computer). You could also launch the script (it is quiet if no error is detected) and then test the return value with `$?`.